libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -69,6 +73,9 @@ export const DealTicket = ({
     },
     [isDisabled, submit, market.decimalPlaces, market.positionDecimalPlaces]
   );
+  const price = isMarketInAuction(market)
```

Comment:
same as in the tooltip, 
we are showing the indicative price if market is in auction and the last traded price otherwise. is this correct? 

We have 0.00 indicative price on this market, and before we were displaying the last traded price ~19,708.74 
What should we do? What would be the correct estimated uncrossing price?

Also, does this apply if market is in Auction in general or just in Monitoring Auction?  @PeteWilliams  

<img width="1177" alt="Screenshot 2022-09-29 at 20 47 39" src="https://user-images.githubusercontent.com/16125548/193119470-75150bf2-575c-4878-8580-16035def2f39.png">

Reply:
I can't make the assumption on the frontend that 0.000 is not a valid price and show `Not available`. this is what is coming from the backend.